Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM python:3.9.0
+WORKDIR /app
+COPY requirements.txt ./
+RUN pip install -r requirements.txt
+COPY . .
+RUN ["python", "harc_game_web/manage.py", "makemigrations"]
+RUN ["python", "harc_game_web/manage.py", "migrate"]
+RUN ["python", "harc_game_web/manage.py", "loaddata", "example_db.json"]
+CMD ["python", "harc_game_web/manage.py", "runserver", "0.0.0.0:8000"]
```

Comment:
could we use `make`s from Makefile?

Reply:
Surely that would be nice! We would need to have `make` in the Docker container though, which I don't think exists in the `python` image (need to check).

Reply:
Actually it looks like `make` is available, so let me have another look at it. Thanks @tqZK for the suggestion! 👏